src/vm/gcenv.ee.cpp

```diff
@@ -932,40 +934,48 @@ void GCToEEInterface::StompWriteBarrier(WriteBarrierParameters* args)
         
         g_lowest_address = args->lowest_address;
         VolatileStore(&g_highest_address, args->highest_address);
```

Comment:
While you're at it, the `WriteBarrierOp::Initialize` path does not need the above VolatileStore or FlushProcessWriteBuffers - no managed code is running and there is only one thread performing EE initialization.